plugins/coverbrowser.koplugin/main.lua

```diff
@@ -226,152 +225,155 @@ function CoverBrowser:addToMainMenu(menu_items)
                         callback = function()
                             self:setupCollectionDisplayMode("list_image_filename")
                         end,
-                        separator = true,
                     },
                 },
-                separator = true,
             },
-            -- Misc settings
+        },
+    }
+
+    -- add Mosaic / Detailed list mode settings to File browser Settings submenu
+    -- next to Classic mode settings
+    if menu_items.filebrowser_settings == nil then return end
+    table.insert (menu_items.filebrowser_settings.sub_item_table, 4, {
+        text = _("Mosaic / Detailed list modes settings"),
+        separator = true,
+        sub_item_table = {
             {
-                text = _("Mosaic and detailed list settings"),
+                text = _("Items per page"),
+                help_text = _([[This sets the number of files and folders per page in non-'classic' display modes.]]),
```

Comment:
I find non-classic a bit awkward.
```suggestion
                help_text = _([[This sets the number of files and folders per page in display modes other than classic.]]),
```